InstallModuleFromGitHub.psm1

```diff
@@ -2,9 +2,9 @@ function Install-ModuleFromGitHub {
     [CmdletBinding()]
     param(
         $GitHubRepo,
-        $Branch = "master",
+        $Branch = 'main',
```

Comment:
This is a breaking change – probably better to put this in a separate PR so the Uri fix below can be published without having to wait for a 2.x release?